lib/pages/chat/events/message.dart

```diff
@@ -453,44 +422,95 @@ class Message extends StatelessWidget {
                     ),
                   ),
                 ),
-              row,
+              GestureDetector(
+                onLongPress: () => controller.selectMode ? onSelect!(event) : null,
+                onTap: () => controller.selectMode ? onSelect!(event) : null,
+                child: Center(
+                  child: Padding(
+                    padding: EdgeInsets.only(
```

Comment:
why not make `selected` outside

Reply:
- When onLongPress in message enable select mode. We can select outside-area messages. Khi disable select mode we only onLongPress inside message.